code/modules/modular_computers/file_system/programs/security/penalmechs.dm

```diff
@@ -0,0 +1,128 @@
+/datum/computer_file/program/penal_mechs
+	filename = "penalmechs"
+	filedesc = "Remote Penal Monitoring"
+	program_icon_state = "security"
+	extended_desc = "This program allows monitoring and control of active penal miner mechs."
+	required_access_run = access_armory
+	required_access_download = access_armory
```

Comment:
Download access migth be higher than dolwnload access to prevent regular uses from getting such program so easily. Tho armory access is lucrative.

Reply:
Going to keep this as `access_armory` based on Geeves' outline for this feature. He wanted the Warden to be able to put it on their tablet.